jetty-core/jetty-server/src/main/config/etc/jetty-graceful.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<!-- =============================================================== -->
+<!-- Mixin the Graceful Handler                                      -->
+<!-- =============================================================== -->
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="insertHandler">
+    <Arg>
+      <New id="StatsHandler" class="org.eclipse.jetty.server.handler.GracefulHandler" />
```

Comment:
Do we want to keep the id as `StatsHandler` or was that overlooked?

Reply:
Good catch. fixing it.